src/project-manager.ts

```diff
@@ -359,7 +354,7 @@ export class ProjectManager implements Disposable {
 				);
 			})
 			// Use same scheme, slashes, host for referenced URI as input file
-			.map(filePath => url.format({ ...parts, pathname: filePath.split(/[\\\/]/).map(encodeURIComponent).join('/'), search: undefined, hash: undefined }))
+			.map(filePath => util.path2uri('', filePath))
```

Comment:
This changes the behaviour I noted in the comment above, which is using the same schems, slashes and host. Instead this now becomes a `file://` URI without host, even if the input was something different. I am trying to decouple the LS from this implication that everything is working with localhost `file://` URIs, this is a step backwards.